src/form/acp.jl

```diff
@@ -984,17 +979,17 @@ function constraint_mc_load_current_delta(pm::AbstractUnbalancedACPModel, nw::In
     vm_a, vm_b, vm_c = var(pm, nw, :vm, load_bus_id)
     va_a, va_b, va_c = var(pm, nw, :va, load_bus_id)
     # v_xy = v_x - v_y
-    vre_xy(vm_x, va_x, vm_y, va_y) = JuMP.@NLexpression(pm.model, vm_x*cos(va_x)-vm_y*cos(va_y))
-    vim_xy(vm_x, va_x, vm_y, va_y) = JuMP.@NLexpression(pm.model, vm_x*sin(va_x)-vm_y*sin(va_y))
+    vre_xy(vm_x, va_x, vm_y, va_y) = JuMP.@expression(pm.model, (vm_x*cos(va_x))-(vm_y*cos(va_y)))
+    vim_xy(vm_x, va_x, vm_y, va_y) = JuMP.@expression(pm.model, (vm_x*sin(va_x))-(vm_y*sin(va_y)))
```

Comment:
So really just a question: you've added all the extra brackets everywhere just to be safe? They really shouldn't be necessary.

Reply:
We agree that we don't think they should be necessary. This was just part of our debugging where we were trying to figure out if extra parentheses were needed or it was adding cancelling minus signs in some equations. I suspect the latter, we will have to selectively revert these changes to double check.